src/testdrive/src/error.rs

```diff
@@ -47,7 +48,7 @@ pub enum Error {
     },
     General {
         ctx: String,
-        cause: Option<Box<dyn StdError + Send + Sync>>,
+        causes: Vec<DynError>,
```

Comment:
I've been kind of hoping to switch testdrive over to anyhow, which this inhibits a bit. I'm not sure that there's a better option, but wanted to get your gears turning.

Reply:
I support that! I can't think of a much better way to do this, either.